docs/tutorials/excel-tutorial-create-custom-functions.md

```diff
@@ -171,61 +171,48 @@ Integrating data from the Web is a great way to extend Excel through custom func
-
-    ```js
-    /**
-    * Streams real time stock price
-    * @customfunction 
-    * @param {string} ticker Stock symbol
-    * @param {CustomFunctions.StreamingInvocation<number>} invocation
-    */
-    function stockPriceStream(ticker, invocation) {
-        var updateFrequency = 1000 /* milliseconds*/;
-        var isPending = false;
-
-        var timer = setInterval(function() {
-            // If there is already a pending request, skip this iteration:
-            if (isPending) {
-                return;
-            }
-
-            //Note: In the following line, replace <YOUR_TOKEN_HERE> with the API token that you've obtained through your IEX Cloud account.
-            var url = "https://cloud.iexapis.com/stable/stock/" + ticker + "/quote/latestPrice?token=<YOUR_TOKEN_HERE>"
-            isPending = true;
-
-            fetch(url)
-                .then(function(response) {
-                    return response.text();
-                })
-                .then(function(text) {
-                    invocation.setResult(parseFloat(text));
-                })
-                .catch(function(error) {
-                    invocation.setResult(error);
-                })
-                .then(function() {
-                    isPending = false;
-                });
-        }, updateFrequency);
-
-        invocation.onCanceled = () => {
-            clearInterval(timer);
-        };
+The `getStarCount` function returns the number of stars a repository has at a specific moment in time, but custom functions can also retrieve or display data that is ever-changing. These functions are called streaming functions and they must include an `invocation` parameter which refers to the content inside of a given cell. This is so that the contents of the cell can be changed in real-time.  
+
+In this new function, we will be making a spinner out of ASCII characters. The spinner will "spin" by displaying a different value every 300 milliseconds. In the event that spinning needs to be cancelled, the contents of the cell will change to display an error message.
+
+1. In the **starcount** project, add the following code to **./src/functions/functions.js** and save the file.
+
+```JS
+/**
+ * Displays a spinner.
+ * @customfunction
+ * @param {CustomFunctions.StreamingInvocation<string>} invocation Custom
+function invocation.
+ */
+function spin(invocation) {
+    const spinIcons = "|/-\\";
+    var i;
+    if (i == undefined) {
+        i = 0;
     }
-    CustomFunctions.associate("STOCKPRICESTREAM", stockPriceStream);
-    ```
-    
-    The `CustomFunctions.associate` code associates the `id` of the function with the function address of `stockPriceStream` in JavaScript so that Excel can call your function.
-    
+    const timer = setInterval(function () {
+        i++;
+        if (i >= spinIcons.length) {
+            i=0;
+        }
+        invocation.setResult(spinIcons[i]);
+    }, 300);
```

Comment:
I had to slow this down to 500 for online to work correctly. I'm thinking we should use a different example since there seems to be variance around this technique.

Reply:
Actually can we use the clock example here? Simple and will work.

Reply:
Will do!